modules/rosa-operator-config.adoc

```diff
@@ -16,25 +17,34 @@ When using a {hcp-title} cluster, you must create the Operator IAM roles that ar
 
 .Procedure
 
+* Set your prefix name to an environment variable using the following command:
```

Comment:
It looks like you're adding a new step that has to come before the next step. So these should be changed to ordered steps.